src/components/dialogs/load-creation-dialog.js

```diff
@@ -216,35 +220,52 @@ const LoadCreationDialog = ({
         setDialogSearchOpen(true);
     };
 
+    const addSuffixIfNecessary = (loadId, i) => {
+        return fetchEquipmentExists(
+            studyUuid,
+            selectedNodeUuid,
+            'loads',
+            loadId + '(' + i + ')'
+        ).then((response) => {
+            if (response.status === 404) {
+                return loadId + '(' + i + ')';
+            } else {
+                return addSuffixIfNecessary(loadId, i + 1);
+            }
+        });
+    };
+
     const handleSelectionChange = (element) => {
         let msg;
-        fetchLoadInfos(studyUuid, workingNodeUuid, element.id).then(
+        fetchLoadInfos(studyUuid, selectedNodeUuid, element.id).then(
             (response) => {
                 if (response.status === 200) {
                     response.json().then((load) => {
-                        //For now we don't want to retrieve nor try to set the BusBarSection, users have to select it.
-                        setLoadId(load.id);
-                        setLoadName(load.name);
-                        setLoadType(load.type);
-                        setActivePower(String(load.p0));
-                        setReactivePower(String(load.q0));
-                        setVoltageLevel(
-                            voltageLevelOptions.find(
-                                (value) => value.id === load.voltageLevelId
-                            )
-                        );
-                        setBusOrBusbarSection(null);
-
-                        msg = intl.formatMessage(
-                            { id: 'LoadCopied' },
-                            {
-                                loadId: element.id,
-                            }
-                        );
-                        enqueueSnackbar(msg, {
-                            variant: 'info',
-                            persist: false,
-                            style: { whiteSpace: 'pre-line' },
+                        addSuffixIfNecessary(load.id, 1).then((loadId) => {
+                            setLoadId(loadId);
+                            //For now we don't want to retrieve nor try to set the BusBarSection, users have to select it.
```

Comment:
maybe you could move this comment near 
`setBusOrBusbarSection(null);`
which is the line concern by this comment

Reply:
done